XamarinCommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -74,6 +74,18 @@
     <Compile Include="**/*.shared.*.cs" />
     <None Include="..\LICENSE" PackagePath="" Pack="true" />
     <None Include="..\assets\XamarinCommunityToolkit_128x128.png" PackagePath="icon.png" Pack="true" />
+    <Compile Include="Views\AvatarView\ColorTheme.cs">
+      <SubType></SubType>
+    </Compile>
+    <Compile Include="Views\AvatarView\ColorThemeBase.cs">
+      <SubType></SubType>
+    </Compile>
+    <Compile Include="Views\AvatarView\ColorThemes.cs">
+      <SubType></SubType>
+    </Compile>
+    <Compile Include="Views\AvatarView\IColorTheme.cs">
+      <SubType></SubType>
+    </Compile>
```

Comment:
Should be discarded (These files should have **.shared.** part of the name e.g. ColorTheme.shared.cs)

Reply:
Should I add .shared to the interface `IColorTheme` also?

Reply:
yes, since all of them belongs to shared code, we must add **shared** part.